parsl/tests/test_globus/test_implicit_staging_https.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+import parsl
+from parsl.app.app import App
+from parsl.data_provider.files import File
+from parsl.tests.configs.local_threads_globus import config
```

Comment:
It looks like this file hasn't been added.

Reply:
https://github.com/Parsl/parsl/commit/49d9594b2d6ac782a9edf4776c4521db9133a622 has been added by you.

Reply:
(facepalm) Thanks @lukaszlacinski sorry I missed that! 😄